tests/unit/test_match.py

```diff
@@ -102,6 +102,44 @@ def test_match_exclude_dataset():
     assert len(res["results"]) == 0, res
 
 
+def test_match_include_dataset():
+    # When querying Putin
+    query = {"queries": {"vv": EXAMPLE}}
+    # Using only datasets that do not include Putin
+    params = {
+        "algorithm": "name-based",
+        "allow_datasets": ["ae_local_terrorists", "mx_governors"],
```

Comment:
A small improvement might be to use datasets that occur in the test data and a query which occurs in both, but I don't think this test is invalid because of this choice.

Reply:
Do you mean using two datasets where Putin occurs in both of them?

Reply:
In the unit tests for yente, only eu_fsf is actually present. So neither of these datasets are in the index.

Reply:
Gotcha. In that case I think all the tests are valid, though a bit wonky.

Reply:
Sorry for the slow response. Yeah - it looks like the fixtures have fsf and parteispenden, and queries which have a match from both datasets would just be a bit more technically correct, since we can only expect matches from datasets in the index.

Something we're not testing for yet, is the behaviour for a single entity that was merged and occurs in both datasets. If only one of those datasets are allowed by query param, is the entity returned? And if any of its datasets are excluded by query param (but not all), is it excluded?